src/components/atoms/two-step-delete/index.tsx

```diff
@@ -0,0 +1,107 @@
+import clsx from "clsx"
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react"
+import TrashIcon from "../../fundamentals/icons/trash-icon"
+import Spinner from "../spinner"
+import Tooltip from "../tooltip"
+
+type Props = {
+  onDelete: () => void
+  deleting?: boolean
+}
+
+const TwoStepDelete = forwardRef<HTMLButtonElement, Props>(
```

Comment:
praise: Agree w. @pKorsholm!

todo: Though, we seem to have an issue with a fleeing tooltip when button is unarmed 😅 
![image](https://user-images.githubusercontent.com/59018053/186612984-a172c9ae-0345-4c0c-9fca-af9b5ea57893.png)

Reply:
Should be good now 😅